lib/proxy.js

```diff
@@ -139,6 +139,10 @@ class Gateway {
     return this.apiRequest({ uri: `${this.api_url}/instance` });
   }
 
+  getModuleVersions(modules) {
+    return this.apiRequest({ method: 'GET', uri: `https://partners.platformos.com/api/pos_modules?modules=${modules.join(',')}` })
```

Comment:
@nemvalid Do not hardcode portal url. Allow to pass url also as ENV variable. Please look at `lib/portal.js`